src/test/unit/profile-data.test.js

```diff
@@ -483,22 +483,14 @@ describe('upgrades', function() {
     );
     compareProcessedProfiles(upgradedProfile5, afterUpgradeReference);
 
-    const geckoProfile3 = require('../fixtures/upgrades/gecko-3.json');
-    const upgradedGeckoProfile3 = unserializeProfileOfArbitraryFormat(
-      geckoProfile3
+    const serializedOldProcessedProfile6 = require('../fixtures/upgrades/processed-6.json');
+    const upgradedProfile6 = unserializeProfileOfArbitraryFormat(
+      serializedOldProcessedProfile6
     );
-    compareProcessedProfiles(upgradedGeckoProfile3, afterUpgradeReference);
```

Comment:
I'm removing this test because there's no way I can make it pass with the current setup without making different one fail. The problem is that processing used to destroy information about relative startup times of the different processes. This means that the following operations can yield different profiles:
old gecko profile -> old processed profile -> new processed profile
old gecko profile -> new gecko profile -> new processed profile

because of different processStartupTime values for child processes. So I can't use the same reference profile to test both paths.